Update inline-arrays.md (#7208)
dotnet/csharplang: proposals/inline-arrays.md

    WriteLine(a);
}

> foreach (var b in (System.ReadOnlySpan<int>)getBufferAsReadonlyVariable())

Author: ```suggestion
foreach (var b in (System.ReadOnlySpan<int>)getBufferAsWritableVariable())
```

> foreach (var b in (System.ReadOnlySpan<int>)getBufferAsWritableVariable())

Author: ```suggestion
foreach (var b in (System.Span<int>)getBufferAsWritableVariable())
```

}
```

> We will support `foreach` over inline arrays, even if it starts as restricted in `async` methods due to involvment of the `System.ReadOnlySpan` type into the translation.

Author: ```suggestion
We will support `foreach` over inline arrays, even if it starts as restricted in `async` methods due to involvement of the `System.ReadOnlySpan` type into the translation.
```

> We will support `foreach` over inline arrays, even if it starts as restricted in `async` methods due to involvement of the `System.ReadOnlySpan` type into the translation.

Author: ```suggestion
We will support `foreach` over inline arrays, even if it starts as restricted in `async` methods due to involvement of the span types into the translation.
```
